helpers/request.ts

```diff
@@ -310,3 +313,69 @@ export function requestTryAndRepeatUntil<T>(
     }
   );
 }
+
+// Adapted from https://nodejs.org/api/http.html#http_http_HTTP_STATUS
+export enum HTTP_STATUS {
```

Comment:
Cool! Should we rename it to `HttpStatus` to match the way we name other enums ? And usually we use `camelCase` for items in there.

Reply:
Also maybe we should drop things that we don't use yet ? Or at least things that we might never use, I'm thinking about those 100s, or weird stuff like 203,205,206,207,226,...,418,...

Dropping theme will make the enum smaller, and make it easier to find what we want during autocompletion.

Reply:
Well, I'd have argued that camelCase is not the best here, given what we documented [here](https://github.com/SonarSource/sonar-enterprise/blob/master/private/docs/web-ui/index.md#types-and-enums) & everything that is [here](https://www.typescriptlang.org/docs/handbook/enums.html).

Reply:
Ah yeah you are right indeed! Seems like we don't respect well the pascal case inside the enum yet on SonarCloud side!

Reply:
Thanks @philippe-perrin-sonarsource , I'll update that and release again :p

Reply:
@siegfried-ehret-sonarsource Well, it works as is. Might not worth the effort of a release.